conans/util/progress_bar.py

```diff
@@ -61,6 +80,7 @@ def pb_update(self, chunk_size):
     def update(self, chunks, chunk_size=1024):
         for chunk in chunks:
             yield chunk
+            time.sleep(0.01)
```

Comment:
Why? This can have some impact on performance, right? And it was not in the previous PR, why introducing it now?

Reply:
sorry, it was for testing purposes and slipped into there 🤦‍♂️